test/server/admin/clusters_params_test.cc

```diff
@@ -0,0 +1,41 @@
+#include "source/common/buffer/buffer_impl.h"
+#include "source/server/admin/clusters_params.h"
+
+#include "gtest/gtest.h"
+
+namespace Envoy {
+namespace Server {
+
```

Comment:
can you put an anon-namespace inside the Server namespace?

Reply:
Sounds good to me. I'll make the adjustments.